CMSgov/mint-app
# Noref/gen char fixes for change history

## Changes and Description

- Remove a few translations with options that mapped to an empty string
    - this violated the translation's zero string domain (which doesn't allow empty strings)
- Expand the option assertion unit test to verify that there are no options that have an empty string



## How to test this change
1. Navigate to general characteristics page 1. 
2. Answer a question that creates an existing model link as well as other fields. Ensure the data still translates as expected

Also you can verify the unit tests pass to assert that the options are correct.


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
